slamd/materials/processing/strategies/aggregates_strategy.py

```diff
@@ -4,29 +4,32 @@
 
 class AggregatesStrategy(BaseMaterialStrategy):
 
-    def create_model(self, submitted_material):
+    @classmethod
+    def create_model(cls, submitted_material):
         composition = Composition(
-            fine_aggregates=submitted_material['fine_aggregates'],
-            coarse_aggregates=submitted_material['coarse_aggregates'],
+            fine_aggregates=float(submitted_material['fine_aggregates']),
```

Comment:
again, we do not need the conversion; actually it leads to errors when the entry is empty. Please, at all places, change it back to saving simply directly without trying to convert to float.